config/initializers/render_pipeline.rb

```diff
@@ -20,7 +20,27 @@
   config.add_emoji 'pride'
   config.add_emoji 'hot_shit'
 
-  config.render_context :default do |c|
+  config.render_context :default, :v1_bio, :v1_content do |c|
```

Comment:
It truncates at 666 currently. I don't think post content does, however, so we might want to separate those two.